Plugin/Model/Menu/Node/AfterSave.php

```diff
@@ -0,0 +1,83 @@
+<?php
+declare(strict_types=1);
+
+namespace Snowdog\Menu\Plugin\Model\Menu\Node;
+
+use Snowdog\Menu\Api\NodeTranslationRepositoryInterface;
+use Snowdog\Menu\Api\Data\NodeTranslationInterface;
```

Comment:
The class Snowdog\Menu\Api\Data\NodeTranslationInterface is imported but never used